ignite/cmd/scaffold.go

```diff
@@ -109,6 +113,51 @@ func scaffoldType(
 	return nil
 }
 
+func addGitChangesVerifier(cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().AddFlagSet(flagSetYes())
+
+	cmd.PreRunE = func(cmd *cobra.Command, args []string) error {
+		appPath := flagGetPath(cmd)
+
+		gitStatus, _ := getGitStatus(appPath)
```

Comment:
Why is error check omitted here?

Reply:
Can we explicitly check if the errors is related to a non existent repo and only ignore if that's the case?

I believe this can be done inside `areChangesCommitted` func. It could return a nil error if repo do not exists.